css/style.scss

```diff
@@ -477,20 +477,19 @@ $primary-color: #3E9CDC;
 
       &::placeholder {
         padding-left: 7px;
-        @include font(1em, "Lato", black);
+        @include font(1em, "Lato", black, normal);
       }
     }
-
+    input[type="text"]{padding-left: $spacing-md};
     .input-name {
```

Comment:
.input-email and .input-name have a lot of common code. use .input for common code and &-email or &-name for special styles